### PR TITLE
dropdown

### PR DESCRIPTION
**Pull request by okavvada**

### CODE REVIEW

File: insta/insta.html [hunk 1]
```diff
@@ -16,8 +16,12 @@
   <script type="text/javascript" src="insta.js"></script>
 </head>
 <body onload="init()">
-  <header><h1><img id="humburger" src='../images/humburger_button.png'></img>Taxeedee: <span id="subtitle">Travels of Olga and Jack</span></h1></header>
-  <main>
+  <header><h1><img id="humburger" src='../images/humburger_button.png'></img> Taxeedee: <span id="subtitle">Travels of Olga and Jack</span></h1></header>
```
Comment (jackdreilly) [inline insta/insta.html]: add new tags on new lines please.

Comment (jackdreilly) [inline insta/insta.html]: `</img>` are never required, please remove.

File: insta/insta.html [hunk 2]
```diff
@@ -16,8 +16,12 @@
...
+    <main>
+      <span id="dropdown" style="display:none">
+      About Us</br>
```
Comment (jackdreilly) [inline insta/insta.html]: Should the first option be "Home" or "Posts"?

File: insta/insta.js [hunk 3]
```diff
@@ -9,4 +9,13 @@ function init() {
     text.removeClass("collapse");
     text.dotdotdot({}).data("dotdotdot").restore();
   });
-}
\ No newline at end of file
+  $("#humburger").click(function(event) {
+    var humburger = document.getElementById("dropdown");
```
Comment (jackdreilly) [inline insta/insta.js]: document.querySelector("#dropdown") is more modern.  This allows you to have consistent code when selecting either by id "#dropdown" or by class ".thisClass" or by tag type "img", etc.

**Review by jackdreilly — CHANGES_REQUESTED** [review]
Great start! Please consider making changes

File: insta/insta.html (hunk 1 above)
Comment (okavvada) [reply insta/insta.html]: 👍

Comment (okavvada) [reply insta/insta.html]: 👍

File: insta/insta.html (hunk 2 above)
Comment (okavvada) [reply insta/insta.html]: 👍

File: insta/insta.js (hunk 3 above)
Comment (okavvada) [reply insta/insta.js]: 👍

**Review by jackdreilly — APPROVED** [review]
Great!

**Pull request merged**